lazy_tensor_core/lazy_tensor_core/csrc/LazyShapeInference.cpp

```diff
@@ -45,6 +45,7 @@
 
 // #include <functional>
 #include "torch/csrc/lazy/core/shape.h"
+#include "aten/src/ATen/core/Reduction.h"
```

Comment:
Can you remove this header? The staging branch build system has an issue where it would cause error in the CI: https://github.com/pytorch/pytorch/actions/runs/1588571349. Since we are in the process of merging things back to the master, we didn't actually look into why. The way to resolve the issue is trying to include headers only if it's necessary.